mobile-app/app/screens/AppNavigator/screens/Dex/components/ViewPoolDetails.tsx

```diff
@@ -0,0 +1,335 @@
+            testID: 'shares_to_add',
+            themedProps: {
+              light: tailwind('text-mono-light-v2-500'),
+              dark: tailwind('text-mono-dark-v2-500')
+            }
+          }}
+          rhs={{
+            value: getPrecisedCurrencyValue(volume24H),
+            testID: `volume_24h_${pairInfo.displaySymbol}`,
+            usdTextStyle: tailwind('text-sm'),
+            prefix: '$'
+          }}
+          testID={`${pairInfo.displaySymbol}_pool_share_amount`}
+        />
+      </View>
+      <View style={tailwind('mb-4')}>
+        <NumberRowV2
+          lhs={{
+            value: translate('screens/RemoveLiquidity', 'Total liquidity'),
+            testID: 'shares_to_add',
+            themedProps: {
+              light: tailwind('text-mono-light-v2-500'),
+              dark: tailwind('text-mono-dark-v2-500')
+            }
+          }}
+          rhs={{
+            value: getPrecisedTokenValue(pairData.totalLiquidity.usd ?? new BigNumber(0)),
+            testID: `total_liquidity_${pairInfo.displaySymbol}_amount`,
+            usdTextStyle: tailwind('text-sm'),
+            prefix: '$'
+          }}
+          testID={`total_liquidity_${pairInfo.displaySymbol}`}
+        />
+      </View>
+      <NumberRowV2
+        lhs={{
+          value: translate('screens/RemoveLiquidity', 'Tokens in {{token}}', {
+            token: pairData.tokenA.displaySymbol
+          }),
+          themedProps: {
+            light: tailwind('text-mono-light-v2-500'),
+            dark: tailwind('text-mono-dark-v2-500')
+          },
+          testID: 'shares_to_add'
+        }}
+        rhs={{
+          value: mappedPair?.tokenA.reserve ?? 0,
+          testID: `Pooled_${pairData.tokenA.displaySymbol}_${denominationCurrency}`,
+          usdTextStyle: tailwind('text-sm'),
+          usdAmount: getUSDValue(
+            new BigNumber(pairData.tokenA.reserve),
+            pairData.tokenB.symbol
+          )
+        }}
+        testID={`${pairInfo.displaySymbol}_pool_share_amount`}
+      />
+      <NumberRowV2
+        lhs={{
+          value: translate('screens/RemoveLiquidity', 'Tokens in {{token}}', {
+            token: pairData.tokenB.displaySymbol
+          }),
+          testID: `Pooled_${pairData.tokenB.displaySymbol}`,
+          themedProps: {
+            light: tailwind('text-mono-light-v2-500'),
+            dark: tailwind('text-mono-dark-v2-500')
+          }
+        }}
+        rhs={{
+          value: mappedPair?.tokenB.reserve ?? 0,
+          testID: `Pooled_${pairData.tokenB.displaySymbol}_${denominationCurrency}`,
+          usdTextStyle: tailwind('text-sm'),
+          usdAmount: getUSDValue(
+            new BigNumber(pairData.tokenB.reserve),
+            pairData.tokenB.symbol
+          )
+        }}
+        testID={`${pairInfo.displaySymbol}_pool_share_amount`}
+      />
+
+      {pairData?.apr?.total !== undefined && pairData?.apr?.total !== null && (
```

Comment:
![image](https://user-images.githubusercontent.com/13292662/183832529-e17083e9-2f4e-4ffa-b584-e5208b6a8975.png)
Suffix not displaying